utils/soca/fig_gallery/marine_vrfy_display/README

```diff
@@ -0,0 +1,25 @@
+################################################
+# README for index_vrfy_marine.html
+################################################
+
+To use, follow these steps:
+
+----------------------------
+
+1. Login to Hera via X2Go and type on the command line:
+
+        firefox index_vrfy_marine.html
+
+2. In the webpage,
+	a. Enter the path to your /vrfy directory.
```

Comment:
that's not correct anymore. Path to something that comes out of hpss?

Reply:
I added an input box where the user declares if the vrfy figures are in their COMROOT or HPSS. This flag determines the internal path that HTML uses to find and display the figures.